canvas_modules/common-canvas/src/common-properties/controls/passwordfield/passwordfield.jsx

```diff
@@ -49,21 +49,22 @@ class PasswordControl extends React.Component {
 		const validationProps = ControlUtils.getValidationProps(this.props.messageInfo, this.props.tableControl);
 		return (
 			<div className={className} data-id={ControlUtils.getDataId(this.props.propertyId)}>
-				<PasswordInput
-					{...validationProps}
-					autoComplete="off"
-					id={this.id}
-					disabled={this.props.state === STATES.DISABLED}
-					placeholder={this.props.control.additionalText}
-					onChange={this.handleChange.bind(this)}
-					value={value}
-					labelText={this.props.controlItem}
-					hideLabel={this.props.tableControl}
-					light={this.props.controller.getLight() && this.props.control.light}
-					tooltipAlignment="end"
-					showPasswordLabel={showPasswordLabel}
-					hidePasswordLabel={hidePasswordLabel}
-				/>
+				<Layer level={this.props.controller.getLight() && this.props.control.light ? 1 : 0}>
```

Comment:
Could we do this at a higher level?  I see the same code in many controls.  I wonder if the wrapper around the control could do this once?

Reply:
I created a separate issue for this - https://github.ibm.com/NGP-TWC/wdp-abstract-canvas/issues/3704
Because this involves testing all the components again with `light` prop.